### DISCUSSION TITLE
How to delete all reviews of a PR via github action?

### DISCUSSION THREAD

**Discussion by tomh-netbet** [community/community#25785] [Actions]
<p>I would like to implement a git action that, given a draft PR, when clicking “ready for review” button, will dismiss all reviews(that happened during the draft stage…).</p>
<p>I see no dismiss all API for Github, so I would have to list all reviews on the PR, and dismiss them all, one by one.</p>
<p>Any suggestions?</p>

**Comment by tomh-netbet** [comment, marked as answer]
<p>To complete this, I did just what i suggested in the question.<br>
I have listed all reviews for the PR, then used a bash script to run a foreach for every one of them and dismiss the review via the API.</p>
<p>Works in production environment beautifully right now.</p>
<p>EDIT:: I can share the code if anyone needs it, just leave a comment.</p>

**Reply by sharprs83** [reply]
Yes, please. We're trying to lock some things down with this sort of method and if I can avoid re-inventing the wheel, that'd be super helpful.

**Comment by markliederbach** [comment]
<p>Hi <a class="mention" href="/tomh-netbet">@tomh-netbet</a>, would you mind sharing what scripting you used to accomplish this?</p>

**Comment by rwilliams3** [comment]
@tomh-netbet Do you mind sharing your code?
